### PR TITLE
Fix #2795: Add release notes for 1.19.2

### LINKED ISSUE #2795: Release Notes for 1.19.2
* Add http/https URL scheme handlers #2784

### PR DESCRIPTION
**Pull request by srirambv**
## Summary of Changes

This pull request fixes #2795

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [X] Issue is assigned to a milestone (should happen at merge time).

### CODE REVIEW

File: CHANGELOG.md
Comment (kylehickinson) [inline CHANGELOG.md:3]: Does this imply we're skipping `1.19.1` now? Or just merging the hotfix release notes?

Comment (kjozwiak) [reply CHANGELOG.md:3]: @kylehickinson I though `1.19.1` was blocked/rejected by Apple hence doing a `1.19.2`? Or are we planning on releasing both versions? We're merging as we figured `1.19.1` might not get released and we're moving to `1.19.2`? We can revert and make it two separate releases if that's not the case. CCing @anthonypkeane  just in case for visibility.

Comment (kylehickinson) [reply CHANGELOG.md:3]: As far as I am aware, we are planning on releasing both

Comment (iccub) [reply CHANGELOG.md:3]: Dropping 1.19.1 is only a last resort in case we have to make code changes in order to pass the review.
Then we could think about merging both releases

Comment (kjozwiak) [reply CHANGELOG.md:3]: @srirambv sounds like we'll need to split these back into two for now and have an entry for `1.19.1` 👍 Apologies, my fault for suggesting to merge them as I thought we're skipping `1.19.1` and moving to `1.19.2`.

**Review by kjozwiak — CHANGES_REQUESTED** [review]

File: CHANGELOG.md
Comment (srirambv) [reply CHANGELOG.md:3]: split the notes for 1.19.1 and 1.19.2

**Review by iccub — APPROVED** [review]

File: CHANGELOG.md
Comment (kjozwiak) [inline CHANGELOG.md]: nit: believe it should be HTTP & HTTPS (capitalized).
* Example: https://en.wikipedia.org/wiki/HTTPS

**Review by kjozwiak — CHANGES_REQUESTED** [review]

**Comment by kjozwiak** [comment]
@srirambv lets not merge until we know 100% re: if we're actually releasing `1.19.1` or going straight into `1.19.2`.

File: CHANGELOG.md
Comment (kjozwiak) [inline CHANGELOG.md]: There's a trailing `sss` that was accidentally added at the end of the URL.

**Review by kjozwiak — CHANGES_REQUESTED** [review]

**Comment by kjozwiak** [comment]
LGTM 👍 We'll need to wait and see that the end result re: `1.19.2` is re: Apple before merging.

**Review by kjozwiak — APPROVED** [review]

**Pull request merged**
